test/spec/protractor_sync_test.ts

```diff
@@ -277,12 +285,18 @@ describe('Protractor extensions', () => {
         testArea.findVisible('.visible-element');
       }));
 
-      it('works with polledExpect regardless of element finder type', createTest(() => {
+      it('automatically resolves promises returned with polledExpect Protractor and Selenium methods', createTest(() => {
         polledExpect(() => testArea.findElement('.visible-element').getElementFinder().getWebElement().isDisplayed()).toEqual(true);
-        polledExpect(() => testArea.findElement('.visible-element').isDisplayed()).toEqual(true);
+      }));
+
+      it('automatically resolves promises returned with polledExpect Protractor methods', createTest(() => {
         polledExpect(() => testArea.findElement('.visible-element').getElementFinder().isDisplayed()).toEqual(true);
       }));
 
+      it('works with polled Expect and protractorSync element finder methods on its own', createTest(() => {
+        polledExpect(() => testArea.findElement('.visible-element').isDisplayed()).toEqual(true);
+      }));
+
```

Comment:
Not your doing, but these methods don't seem to belong in the findVisible() describe block. None of them are even calling findVisible. :)